src/locales/ca_ES.ts

```diff
@@ -38,9 +38,9 @@ export const ca_ES: I18nType = {
   productVersion: {
     deprecatedWarningProduct: "Aquesta versió del producte ja no està vigent. Els punts d'interacció seguiran sent totalment funcionals fins que aquesta versió sigui retirada.",
     unableToRetrieveDoc: 'No es pot recuperar la documentació',
-    noProductVersionsDetail: 'This has not been translated',
-    noProductVersionsTitle: 'This has not been translated',
-    registerProductVersion: 'This has not been translated'
+    noProductVersionsDetail: 'This App is not registered for any Product Versions',
```

Comment:
suggestion: can we leave an annotation in a comment for these so they are easy to track?
```suggestion
    // @translate
    noProductVersionsDetail: 'This App is not registered for any Product Versions',
```

Reply:
personally I would also go for something like
```
function translationNeeded(str: string) { return str }
```

And then wrap any untranslated strings like
```
noProductVersionsDetail: translationNeeded('This App is not registered for any Product Versions'),
```

because that is harder to make a typo for, but I could go either way

Reply:
I like the idea too, I prepared the change if someone else is okay with it

Reply:
Update from conversation: we should reference the actual `en` strings by importing the en file. Then wrap them in `translationNeeded` if it is a string that we would like a translation for. (can omit `translationNeeded` if we intend to use the `en` string)